app/controllers/message_controller.rb

```diff
@@ -0,0 +1,5 @@
+class MessageController < ApplicationController
+  def random
```

Comment:
- [ ] Nice job so far! Running both frontend and backend apps concurrently indicates request from the frontend is hitting the API endpoint at the backend successfully. However, random greetings are not displayed on the screen as intended. When I click on the Message button on the from the browser, it takes me to another page which has a home button, but no message is displayed on either of the screens.  The browser console shows that CORS is not properly configured. Let's ensure to improve upon this aspect.

**Screen 1**
![image](https://user-images.githubusercontent.com/12745474/228247941-7671296b-3669-401c-916b-b2d9eadf8f3c.png)

**Screen 2**
![image](https://user-images.githubusercontent.com/12745474/228248296-3d205c7e-97a8-492a-bf9d-d9752beb36e1.png)